tests/functional/spec/multiple_answers.spec.js

```diff
@@ -1,23 +1,87 @@
-    expect(browser.getUrl()).to.contain(PersonalDetailPage.pageName);
-    expect($(PersonalDetailPage.firstName()).isFocused()).to.be.true;
+  describe("Given I have completed a questionnaire that has multiple answers per question", () => {
+    beforeEach("Load the questionnaire and answer all questions", () => {
+      browser.openQuestionnaire("test_multiple_answers.json");
+      answerAllQuestions();
+    });
+
+    it("When I am on the summary, Then all answers are displayed", () => {
+      expect($(SubmitPage.textfieldAnswer()).getText()).to.equal("John Doe");
+      expect($(SubmitPage.dateAnswer()).getText()).to.equal("1 January 1995");
+      expect($(SubmitPage.checkboxAnswer()).getText()).to.equal("BMW");
+      expect($(SubmitPage.radioAnswer()).getText()).to.equal("Yes");
+      expect($(SubmitPage.currencyAnswer()).getText()).to.equal("£50,000.00");
+      expect($(SubmitPage.monthYearDateAnswer()).getText()).to.equal("October 2021");
+      expect($(SubmitPage.dropdownAnswer()).getText()).to.equal("Silver");
+      expect($(SubmitPage.unitAnswer()).getText()).to.equal("10,000 mi");
+      expect($(SubmitPage.durationAnswer()).getText()).to.equal("3 years 3 months");
+      expect($(SubmitPage.yearDateAnswer()).getText()).to.equal("2019");
+      expect($(SubmitPage.numberAnswer()).getText()).to.equal("5");
+      expect($(SubmitPage.percentageAnswer()).getText()).to.equal("3%");
+      expect($(SubmitPage.mobileNumberAnswer()).getText()).to.equal("07700900111");
+      expect($(SubmitPage.textareaAnswer()).getText()).to.equal("Fuel type petrol");
+
+      expect($(SubmitPage.ageAnswer()).getText()).to.equal("10");
+      expect($(SubmitPage.ageEstimateAnswer()).getText()).to.equal("This age is an estimate");
+    });
+
+    it("When I click 'Change' an answer, Then I should be taken to the correct page and be able to edit my answer", () => {
+      $(SubmitPage.currencyAnswerEdit()).click();
+      expect(browser.getUrl()).to.contain(AboutYou.url());
+      expect(browser.getUrl()).to.contain(AboutYou.currency());
+      expect($(AboutYou.currency()).isFocused()).to.be.true;
+    });
   });
 
-  it("Given a survey has multiple answers for a question when I save the survey then the summary shows all the answers", () => {
-    browser.openQuestionnaire("test_multiple_answers.json");
-    $(PersonalDetailPage.firstName()).setValue("HAN");
-    $(PersonalDetailPage.surname()).setValue("SOLO");
-    $(PersonalDetailPage.submit()).click();
-    expect($(SubmitPage.firstNameAnswer()).getText()).to.contain("HAN");
-    expect($(SubmitPage.surnameAnswer()).getText()).to.contain("SOLO");
+  describe("Given I have launched a questionnaire that has multiple answers per question", () => {
+    beforeEach("Load the questionnaire", () => {
+      browser.openQuestionnaire("test_multiple_answers.json");
+    });
+
+    it("When I am on the question page, Then all answers should have a label/legend", () => {
```

Comment:
This is strictly not quite right at the moment as checkboxAnswer doesn't. It might in the future, but we don't really want to get in the habit of doing that.

Reply:
Added a comment and also added it to the card which will fix checkbox legend.